packages/errors/src/context.ts

```diff
@@ -33,9 +35,15 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<{
         addresses: string[];
         cause?: Error;
     };
+    [SOLANA_ERROR__INVALID_ADDRESS]: {
+        reason?: string;
+    };
     [SOLANA_ERROR__INVALID_KEYPAIR_BYTES]: {
         byteLength: number;
     };
+    [SOLANA_ERROR__INVALID_SEEDS]: {
```

Comment:
I think `SOLANA_ERROR__INVALID_PROGRAM_DERIVED_ADDRESS` would be more appropriate.

Reply:
†`MALFORMED`